pydra/engine/specs.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, node, attr_type):
         self.field = None
 
     def __getattr__(self, name):
-        if name in self.fields:
+        if name in self.fields or name == "all":
```

Comment:
in general this is fine, but this makes `all` a reserved output name (no interface can have it as an output). could we use `all_` ?